src/Console/Application.php

```diff
@@ -76,15 +74,15 @@ final class Application extends BaseApplication
      */
     private $consoleOutput;
 
-    public function __construct(InfectionContainer $container, string $name = self::NAME, string $version = self::VERSION)
+    public function __construct(InfectionContainer $container)
     {
-        parent::__construct($name, $version);
+        parent::__construct(self::NAME, Versions::getVersion('infection/infection'));
```

Comment:
to be checked in `Versions::getVersion()` works fine within the PHAR. If it does, you can also remove https://github.com/infection/infection/blob/master/box.json.dist#L4

Reply:
it works https://travis-ci.org/infection/infection/jobs/599743124#L812